### PR TITLE
lmp-gateway-image: add jobserv python3 dependencies

### PR DESCRIPTION
**Pull request by mike-scott**
- create lmp-feature-jobserv.inc which contains the jobserv dependencies
- add lmp-feature-jobserv.inc to lmp-gateway-image

Signed-off-by: Michael Scott <mike@foundries.io>

### CODE REVIEW

File: recipes-samples/images/lmp-feature-jobserv.inc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Packages required for ansible compatibility
```
Comment (doanac) [inline recipes-samples/images/lmp-feature-jobserv.inc]: i think that comment is wrong.

Comment (mike-scott) [reply recipes-samples/images/lmp-feature-jobserv.inc]: Woops copy pasta

**Comment by mike-scott** [comment]
Fixed comment pointed out by @doanac

File: recipes-samples/images/lmp-feature-jobserv.inc [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Packages required for jobserv compatibility
+CORE_IMAGE_BASE_INSTALL += " \
+    python3-compression \
+    python3-distutils \
+    python3-json \
+    python3-multiprocessing \
+    python3-netclient \
+    python3-pkgutil \
+    python3-shell \
+    python3-unixadmin \
```
Comment (ricardosalveti) [inline recipes-samples/images/lmp-feature-jobserv.inc]: Only python3-json and python3-multiprocessing are required, was able to run jobserv without the others.

Comment (mike-scott) [reply recipes-samples/images/lmp-feature-jobserv.inc]: done

**Pull request merged**